recipes/tgc/all/test_package/test_package.c

```diff
@@ -0,0 +1,18 @@
+#include <tgc.h>
+
+static tgc_t gc;
+
+static void example_function() {
+    char *message = tgc_alloc(&gc, 64);
+    strcpy(message, "No More Memory Leaks!");
```

Comment:
`"tgc.h"` probably includes it already, but you need to include the `string.h` header for strcpy.